### DISCUSSION TITLE
Synchronizing tracks between arrangement and playtime

### DISCUSSION THREAD

**Discussion by thefoxyouknow** [helgoboss/helgobox#1203] [Ideas]
Firstly, thank you for the updated version of Playtime! Secondly, please add synchronization of track creation, deletion, and editing between the arrangement view and Playtime2. When you create a new track in Playtime2, it is successfully created in the "arrangement view" without any issues. However, if you want to open an existing project and use Playtime2 or create a track in the "arrangement view" and then work with it in Playtime2, you cannot do this. It’s quite inconvenient because you have to recreate all the tracks through Playtime. In short, synchronization from Playtime2 to the "arrangement view" works perfectly, but the reverse doesn’t work at all, which is, in my opinion, the only significant downside of Playtime2 at the moment. Thank you very much!

**Comment by helgoboss** [comment]
I assume you closed this discussion because you found the option?

If not, it's here: Right mouse click the matrix settings column and choose "Insert columns for each selected track" (or something like that).
